### PR TITLE
Create 6-6-advanced-automatic-differentiation_cn.ipynb

### PR DESCRIPTION
**Pull request by SexyCarrots**

### CODE REVIEW

File: docs/source/whitepaper/6-6-advanced-automatic-differentiation_cn.ipynb [hunk 1]
```diff
@@ -0,0 +1,667 @@
...
+    "    tc.array_to_tensor(np.array([1.0, 0, 0]), dtype=\"float32\"),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db2d1484",
+   "metadata": {},
+   "source": [
+    "因此我们得到 $\\frac{\\partial \\psi}{\\partial \\theta_0}$，因为切线在第一个位置取 1，在其他位置取 0。"
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-6-advanced-automatic-differentiation_cn.ipynb:299]: 切线 -> 切向量

File: docs/source/whitepaper/6-6-advanced-automatic-differentiation_cn.ipynb [hunk 2]
```diff
@@ -0,0 +1,667 @@
...
+   ],
+   "source": [
+    "jac_value.shape"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c5a80ec5",
+   "metadata": {},
+   "source": [
```
Comment (refraction-ray) [inline docs/source/whitepaper/6-6-advanced-automatic-differentiation_cn.ipynb:392]: translation

**Comment by SexyCarrots** [comment]
# Updated

**Review by refraction-ray — APPROVED** [review]
LGTM

**Comment by refraction-ray** [comment]
Good job, now we are left with https://github.com/tencent-quantum-lab/tensorcircuit/blob/master/docs/source/tutorials/torch_qml.ipynb and many new English docs which should be refreshed with new .po files

**Pull request merged**